docs/installation.md

```diff
@@ -29,10 +29,11 @@ require_once '/path/to/qingstor-sdk.phar';
 
 ```bash
 git clone git@github.com:yunify/qingstor-sdk-php.git
+cd qingstor-sdk-php && composer install
 ```
 
 2. Require `autoload.php` in your php code
```

Comment:
Should change to `index.php` ? Btw, is it strange to have index.php if the project is not a website.